backend/plugin/db/mysql/sync.go

```diff
@@ -266,6 +266,11 @@ func (driver *Driver) SyncDBSchema(ctx context.Context) (*storepb.DatabaseSchema
 			} else {
 				column.DefaultValue = &storepb.ColumnMetadata_Default{Default: &wrapperspb.StringValue{Value: defaultStr.String}}
 			}
+		} else {
+			// TODO(zp): refactor column default value.
+			if strings.Contains(extra, "auto_increment") || strings.Contains(extra, "AUTO_INCREMENT") {
```

Comment:
Compare upper case.

Reply:
@zll600 Maybe using `ToLower` or `ToUpper` is better